test/WireMock.Net.Tests/FluentMockServerTests.cs

```diff
@@ -76,6 +76,34 @@ public void FluentMockServer_ReadStaticMapping_WithGuidFilename()
             Check.That(mappings.First().Title).IsNullOrEmpty();
         }
 
+        [Fact]
+        public void FluentMockServer_ReadStaticMapping_WithResponseBodyFromFile()
+        {
+            string guid = "00000002-ee28-4f29-ae63-1ac9b0802d87";
+
+            string folder = Path.Combine(GetCurrentFolder(), "__admin", "mappings", guid + ".json");
+            string json = File.ReadAllText(folder);
+
+            string responseBodyFilePath = Path.Combine(GetCurrentFolder(), "ResponseBodyFiles", "responsebody.json");
+
+            dynamic jsonObj = Newtonsoft.Json.JsonConvert.DeserializeObject(json);

```

Comment:
Newtonsoft.Json is not needed here. Please import this namespace in this file.

Reply:
I have moved this to using block and updated the commit. @StefH